pandas/tests/io/json/test_readlines.py

```diff
@@ -13,49 +14,75 @@
 
 from pandas.io.json._json import JsonReader
 
+xfail_json_engine_pyarrow = pytest.mark.usefixtures("json_engine_pyarrow_xfail")
```

Comment:
Great now `xfail_json_engine_pyarrow` and `json_engine_pyarrow_xfail` are not needed anymore since the xfails are directly in the test

Reply:
oh got it how it works now. Will do the refactors.

Reply:
it's very elegant actually.